### ISSUE TITLE
Support for GO！

### ISSUE THREAD

**Issue by PotatoChipking** [from-the-basement/tonbo#42]
## Feature Request
Considering that many KV storage engines such as LevelDB, PebblesDB, and BoltDB are implemented in GO, cause they are widely used.

**Comment by ethe** [comment]
Yes tonbo totally could be a go lib, once we support  dynamic schema declaring, we could start this work immediately.

**Comment by PotatoChipking** [comment]
Okay, hope to be able to support sooner, and if there is anything that can help you, I hope to be able to contribute.

**Comment by KKould** [comment]
> Okay, hope to be able to support sooner, and if there is anything that can help you, I hope to be able to contribute.

Thank you for your suggestion. We are currently mainly investing in performance optimization. Supporting go is not a focus in the short term, but I still think this is a very good suggestion. If you are interested, you can promote the support of this feature : )

**Comment by ethe** [comment]
I will reopen it when we plan this binding, thanks @PotatoChipking !
